contents/docs/surveys/creating-surveys.mdx

```diff
@@ -0,0 +1,64 @@
+2. **API:** Implement your own survey UI and use PostHog for handling logic and events.
+
+### Question type
+
+PostHog supports a number of different questions types. All survey types are available for both popover and API display modes.
+
+Below is a table showing the different questions types:
+
+|                        |                                                                                                       |
+|------------------------|-------------------------------------------------------------------------------------------------------|
+| **Open text**              | ![Open text survey](../../images/docs/surveys/freetext-survey.png)                                     |
+| **Link**                   | ![Link survey](../../images/docs/surveys/link-survey.png)                                              |
+| **Rating - emoji**         | ![Emoji rating survey](../../images/docs/surveys/rating-emoji-survey.png)                              |
+| **Rating - number**        | ![Number rating survey](../../images/docs/surveys/rating-number-survey.png)                            |
+| **Single choice select**   | ![Single choice select survey](../../images/docs/surveys/single-select-survey.png)                      |
+| **Multiple choice select** | ![Multiple choice survey](../../images/docs/surveys/multiple-choice-survey.png)                         |
+
+### Targeting
+
+This specifies the conditions a user must meet in order to be shown the survey. The user must meet **ALL** of the conditions you set to be eligible.
+
+You can target your survey to specific users based on:
+
+- **Linked feature flag:** Whether a user has a specific [feature flag](/docs/feature-flags) enabled. For example, if you're rolling out a new landing page using a feature flag `new-landing-page`, you can gather feedback only from users who have that flag enabled.
+
+- **Page URL contains:** Where the current URL matches or contains a specific string. For example, you can target a survey to only show on `https://posthog.com/docs`, or show on all docs pages by using the `*` wildcard: `https://posthog.com/docs/*`.
+
+- **Selector matches:** Whether a specific element appears on the page with the specified class name or ID. For example, you can target a survey with `#my-button` or `.my-button` selector.
+
+- **User/group properties:** You can target a survey to users who have a specific [user](/docs/product-analytics/user-properties) or [group property](/docs/product-analytics/group-analytics#how-to-set-group-properties). For example, you can target a survey to users who have a property `is_paying=true`.
+
+![Survey targeting options](../../images/docs/surveys/targeting-options.png)
+
+## Launching your survey
+
+Once you've set up your survey, click "Save as draft" to save your survey. Then, when you're ready to launch, click the "Launch" button.
```

Comment:
Should we mention testing surveys while in draft mode? How do they do it? Otherwise what is the point of draft mode.

Reply:
The point of draft mode is to give users a space to prepare their surveys before launch and to support survey creation because creating a survey that launches immediately with no "save" point would likely cause total abandonment. It's the same as having a toggle button that's "enable/disable this survey" just presented differently. I imagine many users will also want to create and review their surveys before launching them with their team.

Why don't we try saying something like

```suggestion
Once you've set up your survey, click "Save as draft" to save your survey. Here you'll be able to review any changes before launching your survey officially. You may also edit, stop, or resume your survey at any time.
```